.github/workflows/rc-snapshot.yml

```diff
@@ -41,6 +41,6 @@ jobs:
         run: yarn build
 
       - name: Deploy snapshot version
-        run: yarn changeset version --snapshot ${{ steps.extract_branch.outputs.branch }} && yarn changeset publish --tag ${{ steps.extract_branch.outputs.tag }} --no-git-tag
+        run: yarn changeset version && yarn changeset publish --tag ${{ steps.extract_branch.outputs.tag }} --no-git-tag
```

Comment:
I left the `--tag` here because otherwise the tag would end up being `rc` of which we have multiple at the same time so having a single `rc` tag seems odd.

I'm not 100% certain changeset wil respect it when in pre mode.